test.js

```diff
@@ -1,19 +1,20 @@
-'use strict';
-var path = require('path');
-var test = require('ava');
-var fn = require('./');
-var cwd = path.join(__dirname, 'fixture');
-var pkgPath = path.join(__dirname, 'package.json');
+import test from 'ava';
+import path from 'path';
+import fn from './';
 
-test('async', function (t) {
-	return fn({cwd: cwd}).then(function (x) {
-		t.is(x.pkg.name, 'read-pkg-up');
-		t.is(x.path, pkgPath);
-	});
+const cwd = path.join(__dirname, 'fixture');
+const pkgPath = path.join(__dirname, 'package.json');
```

Comment:
Because it's passed into the function, I don't think this can be removed.

```js
fn({cwd: '/Users/path/to/my/fixture'});
```

Is off course something completely different then

```js
fn({cwd: 'fixture'});
```